src/base/bittorrent/session.cpp

```diff
@@ -198,7 +199,7 @@ namespace
 
         for (auto i = categories.cbegin(); i != categories.cend(); ++i) {
             const QString &category = i.key();
-            foreach (const QString &subcat, Session::expandCategory(category)) {
+            for (const QString &subcat : copyAsConst(Session::expandCategory(category))) {
```

Comment:
seems no need here

Reply:
or you can just remove all the newly added `copyAsConst` and only add them back when clazy complains :P